common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -508,3 +508,65 @@ def test_studio_settings_page_a11y(self):
         })
 
         self.settings_page.a11y_audit.check_for_accessibility_errors()
+
+
+class AdvancedSettingsTest(StudioCourseTest):
+    """
+    Tests for Advanced Settings tab in studio.
+    """
+    def setUp(self):  # pylint: disable=arguments-differ
```

Comment:
Hmm. I'm confused by this pylint error. Any idea what pylint wants you to do instead?